backend/entities/models.py

```diff
@@ -29,46 +28,50 @@
 class Organization(CreationDeletionMixin):
```

Comment:
Since we do not use a `creation_date` field here anymore, we can get rid of the mixin:

```python
class Organization(models.Models):
```